capsules/src/net/icmpv6/icmpv6.rs

```diff
@@ -0,0 +1,182 @@
+//! ICMPv6 layer of the Tock networking stack.
```

Comment:
Everything public needs to be documented. And the documentation shouldn't assume the reader knows exactly the details of ICMP